system_metrics_collector/src/topic_statistics_collector/constants.hpp

```diff
@@ -25,6 +25,11 @@ namespace topic_statistics_constants
 constexpr const char kMsgAgeStatName[] = "message_age";
 constexpr const char kMsgPeriodStatName[] = "message_period";
 constexpr const char kMillisecondUnitName[] = "ms";
+
+constexpr const char kCollectStatsTopicNameParam[] = "collect_topic_name";
+constexpr const char kDefaultCollectStatsTopicName[] = "/topic";
+
+constexpr const char kPublishStatsTopicNameParam[] = "publish_topic_name";
```

Comment:
I would just use the system metrics default here instead of something new.

Reply:
This is the parameter name. The default is `system_metrics` set [here](https://github.com/ros-tooling/system_metrics_collector/blob/prajaktg/subscriber-topic-stats/system_metrics_collector/src/topic_statistics_collector/subscriber_topic_statistics.hpp#L254).